Makefile

```diff
@@ -3,7 +3,7 @@ CXX ?= clang++
 DBG ?=
 export CXX
 export DBG
-CXXFLAGS = -Wall -Werror -Wextra -std=c++98 -O3 -g3
+CXXFLAGS = -Wall -Werror -Wextra -std=c++98 -g3
```

Comment:
I removed the `-O3` flag while debugging. This flag is brilliant :heart:, and it improves the running speed by a lot. But using it while developing it also means more compiling time and possibly inaccurate debugging.